rules/models/changes/ruleset.go

```diff
@@ -0,0 +1,96 @@
+package changes
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+)
+
+// UpdateOp is an update operation.
+type UpdateOp int
```

Comment:
Move the type def and related methods etc related to `UpdateOp` to `op.go`? Also might be more succinct to just call this `Op` instead of `UpdateOp`.

Reply:
I can move this to its own file.